tests/gate_data.py

```diff
@@ -589,3 +589,20 @@ def OrbitalRotation(phi):
             [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1],
         ]
     )
+
+
+def PSWAP(phi):
+    r"""Phase SWAP gate
+    .. math:: \mathtt{PSWAP}(\phi) = \begin{bmatrix}
+            1 & 0 & 0 & 0 \\
+            0 & 0 & e^{i \phi} & 0 \\
+            0 & e^{i \phi} & 0 & 0 \\
+            0 & 0 & 0 & 1
+        \end{bmatrix}.
+    Args:
+        phi (float): rotation angle :math:`\phi`
+    Returns:
+        array[complex]: unitary 4x4 rotation matrix
+    """
+    e = np.exp(1j * phi)
+    return np.array([[1, 0, 0, 0], [0, 0, e, 0], [0, e, 0, 0], [0, 0, 0, 1]])
```

Comment:
Just to be consistent with the other gate data:
```suggestion
    e = math.exp(1j * phi)
    return math.array([[1, 0, 0, 0], [0, 0, e, 0], [0, e, 0, 0], [0, 0, 0, 1]])
```